src/index.js

```diff
@@ -91,6 +91,18 @@ async function run(octokit, context, token) {
 	endGroup();
 
 	startGroup(`[base] Install Dependencies`);
+
+	let yarnLock = await fileExists(path.resolve(cwd, 'yarn.lock'));
+	let packageLock = await fileExists(path.resolve(cwd, 'package-lock.json'));
```

Comment:
Does this work? It re-declares the same variable as in line 35. I guess my main question is if we can re-declare those variables.

Reply:
nvm, I commented on an old revision.